packages/runner/src/setupServer.js

```diff
@@ -3,21 +3,15 @@ import { StatusCodes } from 'http-status-codes';
 import cors from 'cors';
 import { createServer } from 'http';
 import { Server } from 'socket.io';
-import { dirname, join } from 'path';
-import { fileURLToPath } from 'url';
 import { watchFile } from 'fs';
 import { userBackend } from '../config/paths.js';
 import {
   newBoardGame, applyBoardGameResult, filterBoardGame, getPlayerById, removePlayerById,
   validateBoardGame,
 } from './boardGame.js';
-import require from './require.cjs';
+import requireUtil from './requireUtil.cjs';
```

Comment:
I renamed this, which isn't necessary for this change, but I didn't like the `require` which may be confused with the common js `require`.